dashboard/app/assets/javascripts/videos.js.erb

```diff
@@ -26,6 +39,13 @@ function createVideo(options) {
 //   onClose - actions to take after closing the video dialog, or immediately
 //             if the video isn't shown.
 function showVideoDialog(options, forceShowVideo) {
+
+  // Asyncronously load YouTube's iFrame API
+  var tag = document.createElement('script');
+  tag.src = "https://www.youtube.com/iframe_api";
+  var firstScriptTag = document.getElementsByTagName('script')[0];
+  firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);
+
```

Comment:
do we need to prevent multiple `<script>` tags from getting inserted if this function is called more than once, or is that not a problem?